azure-pipelines/publishSamples.yml

```diff
@@ -0,0 +1,27 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
```

Comment:
I suggest you use this name property: `name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)`

It will name pipeline runs based on that format.